partcad/src/partcad/context.py

```
@@ -899,33 +779,32 @@
         if prj is None:
             pc_logging.error("Package %s not found" % project_name)
             return None
-        pc_logging.debug(
-            "Retrieving %s from %s" % (assembly_name, project_name)
-        )
+        pc_logging.debug("Retrieving %s from %s" % (assembly_name, project_name))
         return prj.get_assembly(assembly_name, params)
 
     def get_assembly(self, assembly_spec, params=None):
```

Ruff (0.8.2)

Missing return type annotation for public function `get_assembly`

(ANN201)